manifests/kerberos.pp

```diff
@@ -28,6 +31,7 @@
   Optional[ String ] $createhostkeytab,  # BASE64 ENCODING OF KRB5 CREATEHOST KEYTAB FILE
   Optional[ String ] $createhostuser,
   Hash               $crons,
+  Hash               $files_remove_setuid,
```

Comment:
I think this should be an array. But as noted in the SVC ticket, let's hold off till later since this might need to be revisited in the future.